### DISCUSSION TITLE
Unable to compile it with QT make

### DISCUSSION THREAD

**Discussion by mykelelinco** [sanny32/OpenModSim#16] [Q&A] [answered]
Hi there,
Seem to be having quite a few issues.
Is it possible for you to put on git a Windows exe file, so can run to (what looks to be a great App).
I'm only say an amateur programmer, so the type(s) of compile errors coming up, i'm not able to resolve.
Cheers.

**Comment by sanny32** [comment, marked as answer]
Hello @mykelelinco. You can download release builds here: https://github.com/sanny32/OpenModSim/releases
